test/jasmine/tests/cartesian_interact_test.js

```diff
@@ -188,6 +188,48 @@ describe('main plot pan', function() {
         .then(done);
     });
 
+    it('should emit plotly_relayouting events during pan interactions', function(done) {
+        var mock = require('@mocks/10.json');
+
+        function _drag(x0, y0, x1, y1, n) {
+            mouseEvent('mousedown', x0, y0);
+            var dx = (x1 - x0) / n;
+            var dy = (y1 - y0) / n;
+            for(var i = 0; i <= n; i++) {
+                mouseEvent('mousemove', x0 + dx * i, y0 + dy * i);
+            }
+            mouseEvent('mouseup', x1, y1);
+        }
+
+        var nsteps = 10; var events = []; var relayoutCallback;
+        Plotly.plot(gd, mock.data, mock.layout).then(function() {
+            // Switch to pan mode
+            modeBar = gd._fullLayout._modeBar;
+            var buttonPan = selectButton(modeBar, 'pan2d');
+            buttonPan.click();
+            expect(buttonPan.isActive()).toBe(true); // switched on dragmode
```

Comment:
Non-blocking, but

```js
var mock = Lib.extendDeep({}, require('@mocks/10.json');
mock.layout.dragmode = 'pan'
```

is a much simpler way to start a test in `dragmode: 'pan'`

Reply:
Thanks @etpinard 

Done in 12658db141231bf1e8bc2606f23342c07dcdda40